package.json

```diff
@@ -41,6 +41,7 @@
     "sinon-as-promised": "^2.0.3",
     "sinon-chai": "^2.7.0",
     "supertest": "^0.15.0",
-    "xunit-file": "0.0.6"
+    "xunit-file": "0.0.6",
+    "nock": "^2.17.0"
   }
```

Comment:
Yeah, seems generally useful, though I'm also fine with leaving it in the os-repo-tool-spec unit test since it's not used anywhere else.